tools/noPullRequest

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+normalized_branch=$(echo $HEAD_BRANCH|sed 'sA[/_]A.Ag')
+sonar_uri="https://sonarcloud.io/dashboard?id=$REPO_NAME&branch=$HEAD_BRANCH&resolved=false" 
+repo_uri="https://repo.kodekonveyor.com/$REPO_NAME/$(echo $PULL_REQUEST_REPO_FULL_NAME |sed 'sA/.*AA')/$normalized_branch"
+echo  "building pull request. Sonar and documentation creation are not executed."
```

Comment:
If they are not executed, what is the meaning of the next 2 lines?

Reply:
The URLs point to the artifacts created by the non-PR build of the same pull request.